packages/types/src/types.ts

```diff
@@ -77,7 +77,7 @@ export namespace SocialNetworkUI {
     /** The init() should setup watcher for those states */
     export interface AutonomousState {
         /** My profiles at current network */
-        readonly profiles: ValueRef<readonly ProfileInformation[]>
```

Comment:
why remove readonly? update it in place won't get the update notification and might be a source of bug